Reland Update I2I & I2S to reflect new Open Source process (#25530) with edits (#26394)
ampproject/amphtml: .github/ISSUE_TEMPLATE/intent-to-ship--i2s-.md

Add any other information that may be relevant in determining if your feature can ship.
-->

## Build checklist

This section describes considerations the implementor should keep in mind while building this technical solution.

- [ ] Tests written
<!--
Have unit tests been written? Integration tests? Validator tests?
Details on unit and integration tests:https://github.com/ampproject/amphtml/blob/master/contributing/getting-started-e2e.md#testing-your-changes
Details on validator tests: https://github.com/ampproject/amphtml/blob/master/contributing/component-validator-rules.md
-->
- [ ] Documentation present
<!--
Has reference documentation been added? Have examples been added? Has an example been made available on amp.dev?
-->
- [ ] Cross platform testing
<!--
Have you tested all browsers and versions that AMP supports? https://amp.dev/support/faq/supported-browsers/
Have you tested across mobile and desktop devices? 
-->

### Considerations checklist

This section describes different cross cutting considerations that the implementor should keep in mind while building this technical solution.

> <!--

Reviewer: Remove since sentence.

Author: I don't understand.

Reviewer: Sorry, sentence jumbled. I meant to just remove this paragraph since we no longer want to remove any consideration.

Retain all the considerations below that are applicable to your issue. If a consideration doesn't apply to you please remove that section.
-->

- #### Accessibility Considerations
<!--

> Enter a short snippet for the accessibility considerations of the design. If none apply, enter n/a including a short sentence if not obvious. If you need assistance, reach out to the UI and Accessibility WG for assistance. https://github.com/ampproject/wg-ui-and-a11y

Reviewer: I think that the second sentence (which I realize was copied from me) could use some word-tweaking.

Author: ```suggestion
Enter a short snippet for the accessibility considerations of this design. If none apply, enter N/A along with a short explanation. If you need assistance, reach out to the UI and Accessibility WG: https://github.com/ampproject/wg-ui-and-a11y
```
